schemas/core/booking.json

```diff
@@ -1,381 +1,149 @@
-        },
-        "MODE_RAIL": {
-          "type": "object"
-        },
-        "MODE_BUS": {
-          "type": "object"
-        },
-        "MODE_FERRY": {
-          "type": "object"
-        },
-        "MODE_CABLE_CAR": {
-          "type": "object"
-        },
-        "MODE_GONDOLA": {
-          "type": "object"
-        },
-        "MODE_FUNICULAR": {
-          "type": "object"
-        },
-        "MODE_TRANSIT": {
-          "type": "object"
-        },
-        "MODE_TRAIN": {
-          "type": "object"
-        },
-        "MODE_TRAINISH": {
-          "type": "object"
-        },
-        "MODE_BUSISH": {
-          "type": "object"
-        },
-        "MODE_LEG_SWITCH": {
-          "type": "object"
-        },
-        "MODE_MAAS_TRIP": {
-          "type": "object"
-        },
-        "MODE_MAAS_PERSONAL": {
-          "type": "object"
-        },
-        "MODE_TAXI": {
-          "type": "object",
-          "properties": {
-            "vehicleId": {
-              "description": "The taxi id (in the taxi number plate on top of the vehicle)",
-              "type": "string",
-              "minLength": 1,
-              "maxLength": 256
-            },
-            "vehicleLocation": { "$ref": "./units.json#/definitions/location" },
-            "vehicleType": {
-              "type": "string",
-              "minLength": 1,
-              "maxLength": 256
-            },
-            "vehicleDescription": {
-              "description": "An arbitrary description of the vehicle, e.g. 'Black Jaguar'",
-              "type": "string",
-              "minLength": 1,
-              "maxLength": 256
-            },
-            "driver": {
-              "description": "Driver information",
-              "type": "object",
-              "properties": {
-                "phone": { "$ref": "./customer.json#/definitions/phone" },
-                "name": {
-                  "description": "Driver name, e.g. 'Sergey' or 'Jane S.'",
-                  "type": "string",
-                  "minLength": 1,
-                  "maxLength": 128
-                },
-                "image": {
-                  "$ref": "./units.json#/definitions/url"
-                }
-              }
-            },
-            "dispatchOrderId": {
-              "type": "string",
-              "minLength": 1,
-              "maxLength": 128
-            },
-            "eta": { "$ref": "./units.json#/definitions/time" },
-            "taxiCenter": {
-              "type": "object",
-              "properties": {
-                "name": {
-                  "type": "string",
-                  "minLength": 1,
-                  "maxLength": 64
-                },
-                "phone": { "$ref": "./customer.json#/definitions/phone" }
-              }
-            }
-          },
-          "additionalProperties": false
-        }
-      },
-      "additionalProperties": true
+        "MODE_WALK": "./metas/MODE_WALK.json",
```

Comment:
'metas' should not be plural. 'meta' is more common usage. In any case, since these are all *modes*, perhaps the folder `modes` would be more semantic.

Reply:
I was thinking that it's a folder that contains different metas schemas, not just MODE_X.

Reply:
It exists in plural form -> https://en.wiktionary.org/wiki/metas#English so I guess that's ok (but for me `metas` also sounds a bit off)

Reply:
I quite like "modes"

Reply:
I don't have strong opinion on this, so if everybody agree to rename the folder to `modes`, i'll change it